jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -363,6 +393,24 @@ def _display_args_changed(self, msg={}):
                 mark.colors = [self.color]
                 mark.fill_opacities = [self.fill_opacity]
 
+    def import_region(self, region):
+        """
+        Import an astropy regions object (or file).

```

Comment:
There is a subtle distinction of Astropy being the project/ecosystem and astropy being the core package.
```suggestion
        Import an Astropy regions object (or file).
```